forms-web-app/jest.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   coverageReporters: ['json', 'html', 'text', 'text-summary'],
   coverageThreshold: {
     global: {
-      branches: 94,
+      branches: 93,
```

Comment:
Drops because - as best I understand this - the total amount of code is less due to the validator logic shrinking as per Marc's change to the AC on the ticket.

Reply:
Disgraceful :)